### ISSUE TITLE
Something strange

### ISSUE THREAD

**Issue by krambriw** [andrewjfreyer/monitor#61]
Hello, great script, been running it for a week or so
However, I have noticed something in my logfile that looks strange
Around 09:37 this morning I left my house when the confidence level was 100
Then my confidence correctly drops from 100 down to zero. But it repeats the same a couple of times, then starting at confidence 90. This must be wrong since I was at that time already far away
I arrived back at 16:43

So the repeated behavior in between is suspicious, the script "thinks" I'm still around. Once it reaches zero, it should not indicate 90-63-45-18 anymore
I think
See below

2018-11-07 09:37:41 100 Walter's iPhone
2018-11-07 09:47:49 90 Walter's iPhone
2018-11-07 09:47:57 63 Walter's iPhone
2018-11-07 09:48:5 45 Walter's iPhone
2018-11-07 09:48:14 18 Walter's iPhone
2018-11-07 09:48:17 0 Walter's iPhone
2018-11-07 09:48:28 90 Walter's iPhone
2018-11-07 09:48:36 63 Walter's iPhone
2018-11-07 09:48:44 45 Walter's iPhone
2018-11-07 09:48:53 18 Walter's iPhone
2018-11-07 09:48:56 0 Walter's iPhone
2018-11-07 09:50:24 90 Walter's iPhone
2018-11-07 09:50:33 63 Walter's iPhone
2018-11-07 09:50:41 45 Walter's iPhone
2018-11-07 09:50:49 18 Walter's iPhone
2018-11-07 09:50:53 0 Walter's iPhone
2018-11-07 09:52:4 90 Walter's iPhone
2018-11-07 09:52:13 63 Walter's iPhone
2018-11-07 09:52:21 45 Walter's iPhone
2018-11-07 09:52:29 18 Walter's iPhone
2018-11-07 09:52:32 0 Walter's iPhone
2018-11-07 16:43:49 100 Walter's iPhone

**Comment by andrewjfreyer** [comment]
What version of the script are you running?

**Comment by krambriw** [comment]
Dear Andrew,
I use 
#VERSION NUMBER
version=0.1.675

Kind regards, Walter

**Comment by andrewjfreyer** [comment]
I would recommend that you try the beta.

**Comment by krambriw** [comment]
Where to find the beta?
git pull just gives the same version I already have

**Comment by gieljnssns** [comment]
`git checkout beta`

**Comment by krambriw** [comment]
Thanks, the beta version provides an error message related to btle when I run it and the scanning stops it seems
I replaced the btle with the official version, scanning starts ok but I do not know if there will be other issues due to that

> mqtt trigger: monitor/scan/DEPART
./support/btle: line 37: syntax error near unexpected token `fi'
./support/btle: line 37: `              fi '
> known device: D0:C5:F3:90:BB:7E will publish to: monitor/Node.RED/D0:C5:F3:90:BB:7E
./monitor.sh: line 577: btle_scanner: command not found
./monitor.sh: line 580: btle_listener: command not found

**Comment by andrewjfreyer** [comment]
Beta is just that - a beta. There will be errors periodically because due to active development.

**Comment by krambriw** [comment]
Dear Andrew,
I understand, no problems,
I found out that the CPU load problem was caused by me running the monitor script with sudo, it worked fine with the regular release but not with the beta. Anyway now running without sudo it runs smoothly. Initially I missed the -r parameter but now I use the -ta and -td instead. I built a logic in Node-RED where I keep track if anyone is still @home or if the house is empty. I use this state information to increase the frequency of arrival scans when the house is empty

Thanks for a great work, it is undoubtedly a very advanced bash project, I assume you are very familiar with bash scripting. Myself, I'm not, I would have looked into writing a multi-threaded Python thing, but that's just me, more convenient with Python

Kind regards, Walter
